gqcp/include/Molecule/Molecule.hpp

```diff
@@ -20,6 +20,8 @@
 
 #include "Molecule/NuclearFramework.hpp"
 
+#include <boost/format.hpp>
```

Comment:
I still don't understand why this line should be included. Is there a way that this code works without this `include`?

Reply:
This include allows for line 142 in Molecule.hpp to be written like it is.